common/src/main/java/dev/ftb/mods/ftbchunks/client/map/Waypoint.java

```diff
@@ -2,28 +2,46 @@
 
 import dev.ftb.mods.ftbchunks.client.WaypointMapIcon;
 
+import java.util.Objects;
+
 /**
  * @author LatvianModder
  */
 public class Waypoint {
 	public final MapDimension dimension;
+	public final int x;
+	public final int y;
+	public final int z;
```

Comment:
Not simpler to just use a Vec3?

Reply:
Probably, but yay for public fields, changing it will need a lot more changes throughout the mod. Definitely a good future cleanup though.